_posts/jasonwang/2016-02-03_jasonwang- Chapter_5_Exercises.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+author: wagerpascal
+title: "The Welcome to List City Sign"
+---
+
+Exercise 1:
+<iframe src="https://trinket.io/embed/python/04b6c55ee4" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I initially was attempting to solve the problem without using lists, but broke from the temptation and decided to use them. I knew that Matlab had a method in which you could concatenate additional objects to a matrix, so I investigated the python version, which is the MatrixName.extend() function. 
+Mildly surprised that I can simply state what things I should concatenate onto my matrix through a function, rather than the Matlab way.
+I also discovered the "sum" function for lists, which greatly simplified how I would go about calculating the totals.
+For the "count", a simple counter was established. Not really intended as most counters, but still good.
```

Comment:
You're right that lists make this much easier.  Make sure you understand how you'd do it with loops though- that's exactly what the list functions are doing for you.